### DISCUSSION TITLE
add base path to the url from directus settings

### DISCUSSION THREAD

**Discussion by DhruvSinghBora** [directus/directus#21161] [Feature Requests]
### Summary

i have to configure haproxy for all url like /admin, /items, /contents and more instead i want to use /directus(/)?(.*) in my haproxy config file 

### Motivation

I am facing difficulty in hosting in directus but other services like activemq are giving this feature 

### Detailed Design

we need to add a functionality in ui or we can add base url setting while configuring compose file like we configure password 

### Must Have:
- A  the base path config opt from docker compose file 
- B

### Should Have:
- C 

### Could Have:
- D  an option in ui to configure base path 

### Won't Have:
- E

### Drawbacks

new have to swith to newer version

### Alternatives

too lengthy process using haproxy 

### Adoption Strategy

dont know

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
